solc/CommandLineInterface.cpp

```diff
@@ -464,6 +464,9 @@ void CommandLineInterface::readInputFiles()
 {
 	solAssert(!m_standardJsonInput.has_value());
 
+	if (m_options.input.noImportCallback)
+		m_universalCallback.resetImportCallback();
```

Comment:
Why this? Would this makes `m_fileReader` a `nullptr` and break on line 479?

Reply:
Nope, this makes `UniversalCallback::m_fileReader` a `nullptr`, and not `CommandLineInterface::m_fileReader` a nullptr, since the latter isn't even a pointer. If we try to fetch the `ReadCallback` via `UniversalCallback` when its `m_fileReader` is null, we'll just return an error result, and that's it.

Reply:
Oh, I see, missed that.